src/main/java/no/fintlabs/portal/oauth/NamOAuthClientService.java

```diff
@@ -89,11 +89,22 @@ public void removeOAuthClient(String clientId) {
 
     public OAuthClient getOAuthClient(String clientId) {
         log.info("Fetching client {}...", clientId);
-        try {
-            return restTemplate.getForObject(NamOAuthConstants.CLIENT_URL_TEMPLATE, OAuthClient.class, idpHostname, clientId);
-        } catch (Exception e) {
-            log.error("Unable to get client {}", clientId, e);
-            throw e;
+        for (int i = 1; true; i++) {
+            try {
+                return restTemplate.getForObject(NamOAuthConstants.CLIENT_URL_TEMPLATE, OAuthClient.class, idpHostname, clientId);
+            } catch (Exception e) {
+                log.error("Unable to get client {}, this was iteration number {}", clientId, i);
```

Comment:
Her svelges den opprinnelige feilmeldingen. Jeg vil gjerne få med e i loggen.

Reply:
Så her legger jeg for eksempel til en ny logg-linje på 97 ("Error: ", e)?

Reply:
Ja, det tror jeg er det enkleste. Du kan kombinere i en linje, men da må du endre string-concaten til f.eks. String.format.